app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  validates_presence_of :first_name
+  validates_presence_of :last_name
+validates :email, :presence => true, :format => { :with => /\A^([^@\s]+)@((?:[-a-z0-9]+.)+[a-z]{2,})$\z/i, :multiline => true }
```

Comment:
I recommend updating this syntax to be in line with the most up to date Ruby syntax. For example, presence: true